### PR TITLE
Create installing-tracking-script.md

### PR DESCRIPTION
**Pull request by Kelvindest**
created installing- tracking- script. md #29

### CODE REVIEW

File: docs/home/installing-tracking-script.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## How to install the tracking script
+
+After installation and setup of the cron jobs you're ready to begin tracking contacts. You need to add a simple piece of JavaScript to the websites for each site you wish
```
Comment (RCheesley) [inline docs/home/installing-tracking-script.md:3]:
```suggestion
After installation and setup of the cron jobs you're ready to begin tracking Contacts. You need to add a piece of JavaScript to the websites for each site you wish
```

I think we should add a link to the cron jobs page here, also Contacts should be capitalised and we try not to use terms like 'simply' (Vale flags as a problem to fix)

File: docs/home/installing-tracking-script.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+to track via Mautic. This is a very simple process and you can add this tracking script to your website template file,
```
Comment (RCheesley) [inline docs/home/installing-tracking-script.md:4]: We don't use terms like 'simple' - we use the Google Technical Writing Style Guide which explains more: https://developers.google.com/style/tone?hl=en

```suggestion
to track via Mautic. This is straightforward process, and you can add this tracking script to your website template file,
```

File: docs/home/installing-tracking-script.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+or install a Mautic integration for the more common Content Management System platforms. Here is an example of the tracking JavaScript which you can access by clicking on 'Tracking Settings' in the global configuration:
```
Comment (RCheesley) [inline docs/home/installing-tracking-script.md:5]:
```suggestion
or install a Mautic integration for the more common Content Management System platforms. Here is an example of the tracking JavaScript which you can access by clicking on 'Tracking Settings' in the Global Configuration:
```

File: docs/home/installing-tracking-script.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+   ```
+    (function(w,d,t,u,n,a,m){w['MauticTrackingObject']=n;
+        w[n]=w[n]||function(){(w[n].q=w[n].q||[]).push(arguments)},a=d.createElement(t),
+        m=d.getElementsByTagName(t)[0];a.async=1;a.src=u;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','https://example.com/mautic/mtc.js','mt');
+
+    mt('send', 'pageview');
+   ```
+   You should change the site URL,  `example.com/mautic` with the URL to your Mautic instance - in the  example provided, but it's recommended to copy
```
Comment (RCheesley) [inline docs/home/installing-tracking-script.md:15]:
```suggestion
   You should change the site URL,  `example.com/mautic` with the URL to your Mautic instance in the example provided, but it's recommended to copy
```

File: docs/home/installing-tracking-script.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+   it from the tracking settings in your instance.
```
Comment (RCheesley) [inline docs/home/installing-tracking-script.md:16]:
```suggestion
   the whole code block from the tracking settings in your Mautic instance.
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
Hi @Kelvindest thank you for making this contribution!

I have done a first pass over it for grammar and Vale related fixes. If you're using VS Code when you're editing, I really recommend installing the Vale extension, as it will highlight the problems in real time and allow you to address them.

I will leave this to @fakela to review from the technical perspective of where the doc should sit in the structure - also we need to add a link for the cron jobs page I think. We should be using the linking extension for that, but I'll let you folks sort that one out!

On the existing page, we do also have a link to the contact monitoring page which we should make sure we include / add if/when the page is available: https://docs.mautic.org/en/contacts/manage-contacts/contact-monitoring

Thanks again for the contribution!

File: docs/home/installing-tracking-script.md (hunk 1 above)
Comment (Kelvindest) [reply docs/home/installing-tracking-script.md]: okay

**Comment by fakela** [comment]
> Hi @Kelvindest thank you for making this contribution!
> 
> I have done a first pass over it for grammar and Vale related fixes. If you're using VS Code when you're editing, I really recommend installing the Vale extension, as it will highlight the problems in real time and allow you to address them.
> 
> I will leave this to @fakela to review from the technical perspective of where the doc should sit in the structure - also we need to add a link for the cron jobs page I think. We should be using the linking extension for that, but I'll let you folks sort that one out!
> 
> On the existing page, we do also have a link to the contact monitoring page which we should make sure we include / add if/when the page is available: https://docs.mautic.org/en/contacts/manage-contacts/contact-monitoring
> 
> Thanks again for the contribution!


@Kelvindest thanks for the PR, the installing tracking script page comes after cron jobs. If you don’t mind, I will move this to where it is supposed to be myself.

**Comment by Kelvindest** [comment]
> 

@fakela thank for your response,i will my do it myself after you have edited it.

@RCheesley thank you for the corrections i will keep that in my mind.

**Pull request closed without merging**